Obsidian/Client.cs

```diff
@@ -607,27 +629,15 @@ internal void SendKeepAlive(DateTimeOffset time)
             return;
         }
 
-        Logger.LogDebug($"Doing KeepAlive ({keepAliveId}) with {Player.Username} ({Player.Uuid})");
+        Logger.LogDebug("Doing KeepAlive ({keepAliveId}) with {Username} ({Uuid})", keepAliveId, Player.Username, Player.Uuid);
```

Comment:
We should look into other uses of the logger that don't use string formatting in this way, but that's out of scope for this PR.